test/test_plot_collection.py

```diff
@@ -70,9 +70,21 @@ def test_remove_nonexistent_plot(self):
             plot_coll.remove("plot_3")
         except KeyError:
             pass
-        assert len(plot_coll.plotters) == 1
 
     def test_add_uninitialised_class(self):
         plot_coll = PlotCollection()
         with self.assertRaises(ValueError):
             plot_coll.add_plotter_source(TestPlotProvider)
+
+    def test_add_list_scalar(self):
```

Comment:
Why is this _scalar? It's unclear what this means since it adds "plot_1" (a string?)

Reply:
Adding a scalar variable (instead of vector variable).   I have updated the test name and added a comment.